CHANGES.md

```diff
@@ -182,9 +201,12 @@ To be released.
 [#1240]: https://github.com/planetarium/libplanet/pull/1240
 [#1242]: https://github.com/planetarium/libplanet/pull/1242
 [#1265]: https://github.com/planetarium/libplanet/pull/1265
+[#1267]: https://github.com/planetarium/libplanet/issues/1267
 [#1268]: https://github.com/planetarium/libplanet/pull/1268
 [#1272]: https://github.com/planetarium/libplanet/pull/1272
 [#1274]: https://github.com/planetarium/libplanet/pull/1274
+[#1275]: https://github.com/planetarium/libplanet/pull/1275
+
```

Comment:
It seems to make the interval between changelogs for each versions, into three new lines. In my memory, it was intended to make the interval into two new lines but [other pull request](#1272) also does and it was occurred during rebasing. Other intervals seems all two lines except for between *0.11.1* and *0.11.0*... Could you remove this line?

```suggestion
```